load/cliptrackextractor.py

```diff
@@ -139,14 +154,15 @@ def _whole_clip_stats(self, clip, frames):
             )
         )
 
+        # GP dont think this matter with dynamic thresh
```

Comment:
GP this might be used for creating the videos.   So if we don't clip the min max then maybe that is why we are getting washed out temperature gradients?